internal/locate/region_request.go

```diff
@@ -514,7 +587,7 @@ func (state *invalidLeader) next(_ *retry.Backoffer, _ *replicaSelector) (*RPCCo
 	return nil, nil
 }
 
-func newReplicaSelector(regionCache *RegionCache, regionID RegionVerID) (*replicaSelector, error) {
+func newReplicaSelector(regionCache *RegionCache, regionID RegionVerID, reqType kv.ReplicaReadType, opts ...StoreSelectorOption) (*replicaSelector, error) {
```

Comment:
I think it is worth commentting that `opts` is currently only effective for follower read.